<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1206</width>
    <height>1096</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0" rowspan="4">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QTextBrowser" name="textBrowser">
      <property name="maximumSize">
       <size>
        <width>100</width>
        <height>25</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="load">
      <property name="text">
       <string>Reload Planet</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="pushButton_SavePlanet">
      <property name="text">
       <string>Save Planet</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="pushButton_ChangePlanet">
      <property name="text">
       <string>Change Planet</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="4">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Spawn Regions</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="regions"/>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>x</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QDoubleSpinBox" name="regionX">
        <property name="minimum">
         <double>-7680.000000000000000</double>
        </property>
        <property name="maximum">
         <double>7680.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>y</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QDoubleSpinBox" name="regionY">
        <property name="minimum">
         <double>-7680.000000000000000</double>
        </property>
        <property name="maximum">
         <double>7680.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Radius</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QDoubleSpinBox" name="regionRadius">
        <property name="maximum">
         <double>512.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Tier</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QSpinBox" name="tier"/>
      </item>
      <item row="6" column="1">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Constant</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QSpinBox" name="constant"/>
      </item>
     </layout>
    </item>
    <item row="3" column="1" colspan="4">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Dynamic Spawns</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QListWidget" name="spawnsList"/>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_StaticSpawns">
        <property name="text">
         <string>Static Spawns</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QComboBox" name="comboBox_StaticSpawns"/>
      </item>
      <item row="3" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" rowspan="3">
         <widget class="QTableWidget" name="tableWidget_StaticSpawns">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>100</height>
           </size>
          </property>
          <column>
           <property name="text">
            <string>X</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Z</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Y</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Heading</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Respawn</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>ParentID</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_InsertStaticSpawn">
          <property name="text">
           <string>Insert</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_RemoveStaticSpawn">
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_StaticSpawnSearch">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1206</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionGeneral_settings"/>
    <addaction name="actionConnect_to_database"/>
    <addaction name="actionConsole"/>
   </widget>
   <widget class="QMenu" name="menuCreature_Manager">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionPreOR_Creatures"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <addaction name="menuEdit"/>
   <addaction name="menuTest"/>
   <addaction name="menuCreature_Manager"/>
  </widget>
  <action name="actionConnect_to_database">
   <property name="text">
    <string>Connect to database</string>
   </property>
  </action>
  <action name="actionConsole">
   <property name="text">
    <string>Console</string>
   </property>
  </action>
  <action name="actionDynamic_Spawn_Area">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Dynamic Spawn Area</string>
   </property>
  </action>
  <action name="actionSingle_Spawn_Creature">
   <property name="text">
    <string>Single Spawn Creature</string>
   </property>
  </action>
  <action name="actionGeneral_settings">
   <property name="text">
    <string>General settings</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Creatures</string>
   </property>
  </action>
  <action name="actionPreOR_Creatures">
   <property name="text">
    <string>PreOR Creatures</string>
   </property>
  </action>
  <action name="actionChange_Planet">
   <property name="text">
    <string>Change Planet</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
